bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -366,8 +366,6 @@ def _validate_chunk_row_in_progress(self, chunk):
         """Helper for :meth:`_validate_chunk`"""
         assert self.state == self.ROW_IN_PROGRESS
         self._validate_chunk_status(chunk)
-        if not chunk.HasField('commit_row') and not chunk.reset_row:
-            _raise_if(not chunk.timestamp_micros or not chunk.value)
```

Comment:
timestamp == 0 is valid, so if not chunk.timestamp_micros is true in this case then we should remove it.